### DISCUSSION TITLE
Just Somethings that aren't yet clear for me

### DISCUSSION THREAD

**Discussion by ArturDomingues** [qiskit-community/qiskit-dynamics#79] [Q&A] [answered]
Hello everybody, I started my masters last semester and I with that I have looked across a lot of frameworks to see what would be best to do some Quantum Machine Learning("I know, I'm not beeing specific here") and at the end, at least for simulations, I choose to use qiskit-dynamics to simulate some NV centers dynamics at my lab here in Brazil. I have a good background on physics, but my computational ("I'm not sure if this is the right word") skills are still in development, I'm making some alterations to the Rabi oscillations code and some others, during this process I stumbled across this line "y0 = DensityMatrix.from_label('0')", but in the DensityMatrix docs I didn't find what '0' represents and I was unsure how to use some pre-treated image files from my lab to determine the DensityMatrix of the NV centers. Thank you guys, I'm a little prolix and that's why such a short question has this much text, I think I really need to clarify everything before sending anything, because I don't really know how to filter what is important in any sort of dialog.

**Comment by DanPuzzuoli** [comment, marked as answer]
Hi Artur,

Thanks for the interest in the package and for starting the first discussion thread :).

The `DensityMatrix` object is part of Qiskit Terra's `quantum_info` submodule, so its documentation can be found in the docs for that package. Specifically, the `from_label` class method is [documented here](https://qiskit.org/documentation/stubs/qiskit.quantum_info.DensityMatrix.from_label.html#qiskit.quantum_info.DensityMatrix.from_label) - it is a convenience method for generating density matrices that are given by tensor products of Pauli eigenstates.

**Reply by ArturDomingues** [reply]
Thank you for the attention and really fast answer, I was reading the wrong docs haha, I think I just found the wrong docs while browsing in the qiskit-dynamics pages. When I finish my implementations, or during the use of it, I think I'll have a lot more questions, but I hope I can help you guys improve some of the tools available here with them (the questions) :). If anything is unclear in my text please let me know, English is not my first language and sometimes I can be unclear (even in portuguese or german or spanish hahah-I really love learning things). The best regards and thanks for the great job with this package it is really perfect (It seems like it was made for quantum physicists to made their lifes easier haha [joke]) for what I`m doing right now

**Reply by DanPuzzuoli** [reply]
Thanks for the kind words! Please don't hesitate to ask - looking forward to answering more questions!

**Reply by 1iyd** [reply]
Hello, I just came into contact with NV color center quantum sensing. I want to know what kind of simulation software is needed to do quantum sensing in this respect. Thank you

**Reply by ArturDomingues** [reply]
I mean for doing quantum sensing you need specific hardware, for simulating what you want there are a lot of options, for now I'm using qutip, but qiskit or qiskit dynamics is good, pennylane is a little more limited but can be used, there are some things in julia too, it all depends on what you are aiming to simulate, do you want to simulate lots of nvs interacting with each other or with what, what kind of sensing you are doing, you should try to play with things, at the beginning of next year my simulation results and codes will probably be available in my github, but there are lots of sources on how to do things here in github and in arxiv

**Reply by ArturDomingues** [reply]
There is this repository which is good https://github.com/QISS-HZB/QuaCCAToo
